packages/garbo/src/resources/darts.ts

```diff
@@ -41,17 +41,26 @@ export const DART_PERKS: string[] = [
   "Deal 25-50% greater damage",
 ];
 
+interface ChoiceOptions {
+  [key: string]: number;
+}
+
 export function highestPriorityOption() {
   const options = availableChoiceOptions();
 
+  // Convert the values of options to numbers
+  const numericOptions: ChoiceOptions = Object.fromEntries(
+    Object.entries(options).map(([key, value]) => [key, Number(value)]),
```

Comment:
TS is yelling you about these types because they're incorrect, not because you're handling the types incorrectly. Look at the signature of the function. The _key_ is a number and the _value_ is a string.